src/pages/room/RoomContext.ts

```diff
@@ -13,10 +13,15 @@ export const RoomContext = React.createContext<RoomContextValue>(null!);
 const useGetterAndSetter = <Value>(value: Value) => {
   const [stateValue, set] = useState(value);
 
-  return {
-    get: () => stateValue,
-    set,
-  };
+  const values = React.useMemo(
```

Comment:
Todo: pretty sure this isn't doing what I want (see earlier todo about not causing rerenders)... I think useSt8 might be the answer.